### DISCUSSION TITLE
Interpolate colors from far pixels?

### DISCUSSION THREAD

**Discussion by ghost** [ImageMagick/ImageMagick#6167] [Help]
### ImageMagick version

6.9

### Operating system, version and so on

Linux Debian

### Description

Hi,
Is it possible to set only a few pixels colors on an image and let the algorithm blend/interpolate the colors between the set pixels?

Please not e that the pixels may be far away (corners of image).
 
Thanks

**Comment by snibgo** [comment]
I'm not sure what you mean.

Perhaps https://imagemagick.org/script/command-line-options.php#sparse-color

**Comment by fmw42** [comment]
See https://imagemagick.org/Usage/canvas/#sparse-color

**Comment by ghost** [comment]
Thanks! Yes it is what I was after, the algorithms don't behave like I expected though but yes, your answers are correct.
Thanks again
